services/web/server/src/simcore_service_webserver/exporter/config.py

```diff
@@ -0,0 +1,17 @@
+import trafaret as T
+
+from aiohttp.web import Application
+from servicelib.application_keys import APP_CONFIG_KEY
+
+CONFIG_SECTION_NAME = "exporter"
+
```

Comment:
Please do not use trafaret. We are placing it by pydantic.
We did a first iteration a while ago (see #1890) and still need to do a second to completely remove it .

Reply:
Just write the schema using pydantic, i.e.

```python
class ExporterSettings(BaseSettings):
     max_uload_file_size: PositiveInt = 10
    downloader_max_timeout_seconds: PositiveInt = 60*60

```

The idea is that every app module "injects" a single instance of these settings in the app. The init and the injection will be done 

See as example how is it done in [scicrunch submodule](https://github.com/ITISFoundation/osparc-simcore/blob/master/services/web/server/src/simcore_service_webserver/scicrunch/submodule_setup.py#L19). 

Notice how the settings class has function to set/get a single instance of settings.

Finally. There i no need to add a section anymore in the input config file.